pyproj/crs/crs.py

```diff
@@ -1200,7 +1200,15 @@ def to_wkt(
         -------
         str
```

Comment:
Mind updating the doc strings as well? Something like this:
```suggestion
        Optional[str]
```